spec/system/visit_index_page_spec.rb

```diff
@@ -35,6 +35,12 @@
     expect(current_path).to eql view_certificate_path(msa_component.component_type, msa_component.id, msa_component.encryption_certificate_id)
   end
 
+  it 'Displays which services a component is used by' do
+    service = create(:service, sp_component: create(:sp_component))
+    visit root_path
+    expect(page).to have_content service.name
```

Comment:
it would be better if this was more specific, and not just looking at the whole page....